src/Room.js

```diff
@@ -319,6 +320,18 @@ function notificationCallback (data) {
     return this.kuzzle.emitEvent('jwtTokenExpired');
   }
 
+  if (data.controller === 'document') {
+    data.type = 'document';
+    data.document = new Document(this.collection, data.result._id, data.result._source);
+    delete data.result;
+  }
+
+  if (data.controller === 'realtime') {
+    data.type = 'user';
+    data.user = {count: data.result.count};
```

Comment:
The initial PR description has been kept for history but the actual implementation matches the one we have aggreed on.

So, no result anymore, but a new field named as the new type property. In this case, it is normal we have a user.